packages/loader/container-loader/src/container.ts

```diff
@@ -833,10 +833,31 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this._resolvedUrl = resolvedUrl;
             await this.connectStorageService();
 
-            // upload blobs here (NYI)
-
-            // post summary here
             if (hasAttachmentBlobs) {
+                // upload blobs to storage
+                assert(!!this.loader.services.detachedBlobStorage, "assertion for type narrowing");
+                const redirectTable: Map<string, string | undefined>
```

Comment:
It would be great to add some comments describing why this flow is needed. I.e. what kind of problem it solves.